packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -133,6 +133,7 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
       _isOffsetHeightDifferent(this._nonStickyContent, this._stickyContentTop) ||
       _isOffsetHeightDifferent(this._nonStickyContent, this._stickyContentBottom) ||
       _isOffsetHeightDifferent(this._nonStickyContent, this._placeHolder) ||
+      _isOffsetWidthDifferent(this._nonStickyContent, this._placeHolder) ||
```

Comment:
Shouldn't these be changed to `this._nonStickyContent.fistElementChild` instead of `this._nonStickyContent`.
(in sync with what you are doing in ` _getNonStickyPlaceholderHeightAndWidth`

Reply:
Whoops, thank you for catching that.  I changed it to firstElementChild; changed the code for `_isOffsetHeightDifferent` and all similar functions to use `HTMLElement` instead of a `RefObject` and added a helper `_getRefHTMLElement` to grab the HTMLElement from the `RefObject`